css/style.css

```diff
@@ -1,24 +1,34 @@
-html {font-size: 20px;  }
+html {
+  font-size: 20px;
+}
+
 img {
-    max-width: 100%;
-    display: block;
-    height: auto;
-    margin: 0 auto;}
+  max-width: 100%;
+  display: block;
+  height: auto;
+  margin: 0 auto;
+}
+
+.container-fluid {
+  padding: 10px;
+}
+
+body {
```

Comment:
this styling has no corresponding body tag. I suggest you add body tags in html for this styling to appear